docs/entities/tokens/non-fungible-token.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "non-fungible-token-metadata",
+  "title": "NonFungibleTokenMetadata",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "token_uri",
+    "name",
+    "description",
+    "image_uri",
+    "image_canonical_uri"
```

Comment:
@asimm241 can we also include the contract principal and txid that deployed the contract in both the NFT and FT payloads?

Reply:
sure